source/features/index.js

```diff
@@ -0,0 +1,187 @@
+import groupBy from 'lodash.groupby';
+import sortBy from 'lodash.sortby';
+
+export const features = {
+	/* GENERAL */
+	keyboardShortcuts: {
```

Comment:
Most options in this file should *not* be configurable imo.

Reply:
You can set `hidden: true` to easily disable any configuration; I think it's a good idea to have all features centralized on this file